src/dragtor/utils/audio_diarization_utils.py

```diff
@@ -0,0 +1,224 @@
+import re
```

Comment:
I feel like all of these functions should be private methods of the `AudioLoader` class - they are not meant to be used on their own or in multiple different contexts